.github/workflows/node.js.yml

```diff
@@ -78,5 +78,8 @@ jobs:
             ${{ runner.os }}-yarn-
       - run: yarn --frozen-lockfile --prefer-offline
       - run: yarn run build
-
-
+      - run: zip -5 -r prebuild-node${{ matrix.node-version }}.zip ./ -x '*.git*' -x 'node_modules/*'
```

Comment:
I would use `tar -cvpzf prebuild-node${{ matrix.node-version }}.tar.gz  --exclude=prebuild-node${{ matrix.node-version }}.tar.gz  --exclude=node_modules  --exclude=.git ` as this preserves permissions such as the executable bit (which we use in `bin/`).

Reply:
Well, I'm open to use tar instead of zip, I just thought that maybe zip is even more well-known than tar but as we're targeting linux-platforms anyway, zip really doesn't make any sense. I'm gonna change it.